lambda-cloudformation-custom-resource/src/main/scala/feral/lambda/cloudformation/CloudFormationCustomResource.scala

```diff
@@ -38,6 +39,12 @@ trait CloudFormationCustomResource[F[_], Input, Output] {
 }
 
 object CloudFormationCustomResource {
+  private implicit def jsonEncoder[F[_]]: EntityEncoder[F, Json] =
+    jsonEncoderWithPrinter(
+      Printer(
+        dropNullValues = true,
+        indent = ""
+      ))
```

Comment:
WDYT about `defaultPrinter.copy(dropNullValues = true)` instead? So we always whatever good defaults are in the http4s encoder.

Also, I wonder if we need this fix also in the encoder for lambdas that return an output, like the http one.

Reply:
I'm not sure about the other lambdas—it's possible, but it's also possible that the nulls should stay in those. Probably needs to be evaluated for each type. I'll switch to the default printer here though.

Reply:
FYI `defaultPrinter` is protected in `CirceInstances` so I'm going to use `Printer.noSpaces` instead.

Reply:
Oh yeah that's fine. Or inherit from `CirceInstances` too maybe. Whatever works.